.gitignore

```diff
@@ -63,5 +64,5 @@ pip-delete-this-directory.txt
 pip-log.txt
 sdist
 target
+user_config.json
```

Comment:
Think added in tests

Reply:
I think this might be from the napari plugin?